### PR TITLE
make Rails 6.1 applicable

### PR DESCRIPTION
**Pull request by ghost**
### Overview
This updates the gem usable with Rails 6.0 or above.

### Confirmation

- [x] rspec cases are green on Actions

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Setup Ruby, JRuby and TruffleRuby
+        uses: ruby/setup-ruby@v1.95.0
+        with:
+          ruby-version: 2.6
+
+      - name: install bundler 1.17.3
+        run: gem install bundler -v 1.17.3
+
+      - name: Install dependencies
+        run: bundle _1.17.3_ install
```
Comment (mingming-lu) [inline .github/workflows/ruby.yml]: It seems to be more maintainable if you set `BUNDLE_VERSION=1.17.3` as an environment variable

**Review by mingming-lu — APPROVED** [review]
LGTM but left a comment

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  BUNDLE_VERSION: 1.17.3
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby, JRuby and TruffleRuby
+        uses: ruby/setup-ruby@v1.95.0
```
Comment (rakumiki) [inline .github/workflows/ruby.yml]: From the [usage](https://github.com/ruby/setup-ruby#usage) in README, it seems `setup-ruby@v1` is used in the cases. Do you have any intention to set a minor version?

File: spec/rails/application/configuration_spec.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 describe Rails::Application::Configuration do
   describe '#cache_store' do
-    let(:configuration) { Rails::Application::Configuration.new }
+    let(:configuration) { Rails::Application::Configuration.new(Pathname.new('')) }
```
Comment (rakumiki) [inline spec/rails/application/configuration_spec.rb:4]: Could you explain why you changed it?

**Review by rakumiki — APPROVED** [review]
LGTM

File: .github/workflows/ruby.yml (hunk 2 above)
Comment (ghost) [reply .github/workflows/ruby.yml]: I have no intention, it just came from my snippet. I'll fix it.

File: spec/rails/application/configuration_spec.rb (hunk 3 above)
Comment (ghost) [reply spec/rails/application/configuration_spec.rb:4]: With Rails 6.1 `Rails::Application::Configuration.new` will call `credentials_available_for_current_env?` (https://github.com/rails/rails/blob/v6.1.5/railties/lib/rails/application/configuration.rb#L423). Without this blank Pathname that method raises NoMethodError.

```
  1) Rails::Application::Configuration#cache_store default 
     Failure/Error: let(:configuration) { Rails::Application::Configuration.new }
     
     NoMethodError:
       undefined method `join' for nil:NilClass
     # /home/user/.rbenv/versions/2.6.9/lib/ruby/gems/2.6.0/gems/railties-6.1.5/lib/rails/application/configuration.rb:424:in `credentials_available_for_current_env?'
     # /home/user/.rbenv/versions/2.6.9/lib/ruby/gems/2.6.0/gems/railties-6.1.5/lib/rails/application/configuration.rb:408:in `default_credentials_content_path'
     # /home/user/.rbenv/versions/2.6.9/lib/ruby/gems/2.6.0/gems/railties-6.1.5/lib/rails/application/configuration.rb:75:in `initialize'
     # ./spec/rails/application/configuration_spec.rb:4:in `new'
```

Comment (rakumiki) [reply spec/rails/application/configuration_spec.rb:4]: Understood.
Root path is nil for the configuration so you pass it to the [initializer](https://api.rubyonrails.org/classes/Rails/Engine/Configuration.html#method-c-new).

**Pull request merged**